### PR TITLE
fix: seeing/hearing other players in multichar

### PR DESCRIPTION
**Pull request by MafewTM**
## Description

Hopefully fixes seeing/hearing other players in multichar as stated in #226 

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My code fits the style guidelines.
- [ ] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/character.lua [hunk 1]
```diff
@@ -44,6 +44,7 @@ local function destroyPreviewCam()
     DestroyCam(previewCam, true)
     RenderScriptCams(false, false, 1, true, true)
     FreezeEntityPosition(cache.ped, false)
+    NetworkConcealPlayer(cache.ped, false)
```
Comment (FjamZoo) [inline client/character.lua:47]: Use cache.playerId and not cache.ped

File: client/character.lua [hunk 2]
```diff
@@ -256,6 +257,7 @@ local function chooseCharacter()
     end
 
     FreezeEntityPosition(cache.ped, true)
+    NetworkConcealPlayer(cache.ped, true)
```
Comment (FjamZoo) [inline client/character.lua:260]: Use cache.playerId and not cache.ped

**Review by FjamZoo — CHANGES_REQUESTED** [review]

**Review by FjamZoo — APPROVED** [review]

**Review by YvanCywan — APPROVED** [review]

**Comment by FjamZoo** [comment]
BTW this should probably have proper testing due to the fact we use OAL which normally needs every single parameter inside of natives filled out to work properly, and it looks like: https://docs.fivem.net/natives/?_0xBBDF066252829606

Has a 3rd parameter, which could cause issues making this "not work"

**Comment by BerkieBb** [comment]
> BTW this should probably have proper testing due to the fact we use OAL which normally needs every single parameter inside of natives filled out to work properly, and it looks like: https://docs.fivem.net/natives/?_0xBBDF066252829606
> 
> Has a 3rd parameter, which could cause issues making this "not work"

That's actually not OAL that requires it, just the vscode extension. OAL only requires you to not use vector unpacking. You can leave last arguments empty because they always have a default value, usually it's false

**Comment by FjamZoo** [comment]
> > BTW this should probably have proper testing due to the fact we use OAL which normally needs every single parameter inside of natives filled out to work properly, and it looks like: https://docs.fivem.net/natives/?_0xBBDF066252829606
> > Has a 3rd parameter, which could cause issues making this "not work"
> 
> That's actually not OAL that requires it, just the vscode extension. OAL only requires you to not use vector unpacking. You can leave last arguments empty because they always have a default value, usually it's false

Sounds good i will merge then

**Pull request merged**